build.savant

```diff
@@ -14,7 +14,7 @@
  * language governing permissions and limitations under the License.
  */
 
-project(group: "com.inversoft.cleanspeak", name: "cleanspeak-containers", version: "4.1.0", licenses: ["ApacheV2_0"]) {
+project(group: "com.inversoft.cleanspeak", name: "cleanspeak-containers", version: "4.1.1", licenses: ["ApacheV2_0"]) {
```

Comment:
Same comment from `-app`, this seems like it should be `4.2.0`.

Reply:
I could see that. I suppose I didn't think of this as a feature that would be announced publicly yet thus 4.1.1 but I suppose 4.2.0 is right, even if not announced.